explorer/src/serializers/jobRunSerializer.ts

```diff
@@ -20,7 +20,7 @@ export const BASE_ATTRIBUTES = [
 
 export const chainlinkNode = {
   ref: 'id',
-  attributes: ['name']
+  attributes: ['name', 'url']
```

Comment:
@rupurt not sure what's going on with https://github.com/smartcontractkit/chainlink/pull/1481#issuecomment-515250811 
I was thinking this was enough to add it into serialization. Did you test this out on a fresh new compile?

Reply:
@NavyAdmiral I can confirm it's now working on the latest changes. Nice one 💯